src/About/About.module.css

```diff
@@ -1,3 +1,46 @@
-.title {
+.title,
+.content {
   color: #3f51b5;
 }
+
+ol {
+  margin: 0;
+}
+
+.error {
+  color: red;
+}
+
+.bio {
+  margin-bottom: 20px;
+}
+
+.repo_list {
+  color: red;
+  font-size: 20px;
+}
+
+.link {
+  background-color: #3f51b5;
+  border-radius: 5px;
+  padding: 6px 16px;
+  color: #fff;
+  display: block;
+  width: 70%;
+  margin: 10px auto;
+  text-align: center;
+  transition: all .3 ease;
+}
+
+.link:hover {
+  background-color: #47549e;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, .3);
+}
+.link:active {
+  background-color: transparent;
+  color: #3f51b5;
+}
+
+.wrap {
+  text-align: center;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=5973330)